configuration/pih/htmlforms/mentalHealth.xml

```diff
@@ -793,7 +793,76 @@
                 </p>
 
             </div>
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.TraumaticEvent"/>
+                    </label>
+                    <obs conceptId="PIH:12362" style="radio" answerConceptIds="PIH:YES,PIH:NO" answerCodes="yes,no" answerSeparator=""/>
+                </p>
+
+            </div>
+        </div>
+
+        <div class="section-container" id="radio_button_container">
+
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.PoorIntrospection"/>
+                    </label>
+                    <obs conceptId="CIEL:166051" style="radio" answerConceptIds="PIH:YES,PIH:NO" answerCodes="yes,no" answerSeparator=""/>
+                </p>
+
+            </div>
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.ThoughtContent"/>
+                    </label>
+                    <obs conceptId="PIH:15034" style="radio" answerConceptIds="PIH:NORMAL,PIH:ABNORMAL"  answerSeparator=""/>
+                </p>
+
+            </div>
+        </div>
+
+        <div class="section-container" id="radio_button_container">
+
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.suicidal"/>
```

Comment:
The label and obs don't match.  Label is Suicidal but the obs/question is "Thought content".  yes/no are not the right answers.

Reply:
What should be the type, @lnball ?

Reply:
@Dutervil I don't think that "thought content" has a yes/no answer.  It's usually asked with abnormal thoughts as answers (ie. delusions, negativism, paranoid delusion, denies suicidal thoughts, denies homicidal thoughts.).  See this example for Liberia.  
![image](https://github.com/PIH/openmrs-config-pihemr/assets/1164849/7b23bd98-c1a3-4337-97ca-2def377a085c)
If your users want a simple radio, I would recommend answers of normal/abnormal.